webui/react/src/components/HyperparameterSearchModal.module.scss

```diff
@@ -14,9 +14,6 @@
     :global(.ant-modal-confirm-btns) {
       display: none;
     }
-    :global(.ant-form-item-explain) {
```

Comment:
i think we wanna hide error message only in HP search, but wanna show it somewhere else. 
This issue cause this layout issue

<img width="1014" alt="image" src="https://github.com/user-attachments/assets/1093ef4a-4e7e-4ab0-b149-bb3f908e3082">